bw1879/project.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Thistle"
+author:
+  - Brad Windsor <bwindsor22@gmail.com>
+---
+
+> In what follows instructions (including this one) are in blockquotes, while
+> text not in blockquotes is a sample. You should of course replace the title
+> author metadata that shows up above with appropriate values.
+>
+> You should start your proposal by stating a problem statement or an overall
+> goal  as below.
+
+For our Big Data and Machine Learning final project we plan to develop a
+framework for writing distributed quantum computing applications.
```

Comment:
Can you please remove the instructions I put in blockquotes, and also fix up this initial sentence since you are (thankfully) not working on distributed quantum computing.

Reply:
Excellent, completed both. Also added @kc2853.